mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/mapping/definition/programmatic/TypeMappingIndexedStep.java

```diff
@@ -38,10 +41,28 @@
 	TypeMappingIndexedStep enabled(boolean enabled);
 
 	/**
+	 * Define a routing binder, responsible for creating a bridge.
+	 * To pass some parameters to the bridge,
+	 * use the method {@link #routingBinder(RoutingBinder, Map)} instead.
+	 *
+	 * @param binder A {@link RoutingBinder} responsible for creating a bridge.
+	 * @return {@code this}, for method chaining.
+	 * @see Indexed#routingBinder()
+	 */
+	default TypeMappingIndexedStep routingBinder(RoutingBinder binder) {
+		return routingBinder( binder, Collections.emptyMap() );
+	}
+
+	/**
+	 * Define a routing binder, responsible for creating a bridge.
+	 * With this method it is possible to pass a set of parameters to the binder,
+	 * so that they can be used by the bridge.
```

Comment:
```suggestion
	 * With this method it is possible to pass a set of parameters to the binder.
```

Parameters don't have to be used by the bridge, necessarily. They can just control mapping options (is this field sortable, ...).